content/de/style-guide/_index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Style Guide
```

Comment:
@iamNoah1 
Do we need to localize the title ? :) 

Anyway all configuration look good to me ;) I am ready to approve.

Reply:
IMHO we can leave it like it is. There are a lot of english words that are used in german language as well and sound strange if we force-translate them ^^. What you think @DaveVentura, @CathPag?